odoo/addons/openupgrade_records/lib/apriori.py

```diff
@@ -75,4 +75,6 @@
     'ir.actions.report.xml': 'ir.actions.report',
     'stock.pack.operation': 'stock.move.line',
     'stock.picking.wave': 'stock.picking.batch',
+    # OCA/partner-contact
+    'res.partner.sector': 'res.partner.industry',
```

Comment:
How about the pre_init_hook of the OCA module?

Reply:
The problem is that Odoo already creates this model on their base module, so it will conflict. Maybe we should only leave that old model to die? I'm thinking it's the best solution, as this one is not complete:

* Model and fields XML will still point to other module (not `base`).
* There's no real advantage of making this, as records are moved other way.

Reply:
Ah, I see. Don't you want to add a full migration in the base modules' migration scripts that is only run if the old sector module was installed?

Reply:
Interesting question @StefanRijnhart I will do that.